src/Core/Integration/TSDFVolume.h

```diff
@@ -33,6 +33,12 @@
 
 namespace three {
 
+enum class TSDFVolumeColorType {
+    None = 0,
+    RGB8Bit = 1,
+    Gray32Bit = 2,
```

Comment:
I am not very confident about this names. You can suggest any.

Reply:
@syncle maybe you could adapt something like similar to the opencv's `Mat` types e.g. `CV_64FC1`, `CV_8UC3`, etc... here is a list http://ninghang.blogspot.com/2012/11/list-of-mat-type-in-opencv.html . Thus in the current case you could have something like `3D_8UC3` (or `3D_8U` without the channel info) and `3D_32FC1` (`3D_32F`)  or if this is getting too specific considering that in your case you have only two cases to worry about, you could go for something like `RGB8` (or with small `rgb8`) and equivalently `RGB32` (or `rgb32`) or `GRAY32` (or `gray32`).

Reply:
Good suggestion. How do you think? @qianyizh?

Reply:
I think it is a good suggestion. Can you do it in this PR?

Reply:
Sure. Will do.